packages/fxa-auth-server/lib/oauth/db/mysql/index.js

```diff
@@ -540,7 +525,7 @@ MysqlStore.prototype = {
    * @param {String} uid User ID as hex
    * @returns {Promise}
    */
-  getAccessTokensByUid: async function getAccessTokensByUid(uid) {
+  _getAccessTokensByUid: async function _getAccessTokensByUid(uid) {
```

Comment:
Should this be `async function getAccessTokensByUid`?

Reply:
i don't think so. i renamed all the functions that `db/index.js` implements so that the part that copies the functions from here doesn't overwrite them.

Reply:
oic... i wasn't consistent in renaming both sides on all the functions. i'll fix that